roles/vald/defaults/main.yml

```diff
@@ -1,16 +1,18 @@
 ---
 # Maintains directories and paths
-vald_home_dir: /opt/{{ service_identifier }}
+vald_home_dir: "{{ vald_axelar_home }}"
 
 vald_bin_dir: "{{ vald_home_dir }}/bin"
-vald_env_dir: "{{ vald_home_dir }}/env"
+vald_env_dir: "{{ vald_home_dir }}/environment"
+vald_key_dir: "{{ vald_home_dir }}/keyring-file"
```

Comment:
we might not have to create this vald_key_dir, it will get created when we will import the keys